frontend/src/models/propertyDefinitionsModel.ts

```diff
@@ -144,6 +145,14 @@ const constructValuesEndpoint = (
         eventParams += `&event_name=${eventName}`
     }
 
+    // Add property filters
+    if (properties?.length) {
+        for (const prop of properties) {
+            const values = Array.isArray(prop.values) ? prop.values : [prop.values]
+            eventParams += `&properties_${prop.key}=${encodeURIComponent(JSON.stringify(values))}`
+        }
+    }
```

Comment:
I'm not a big fan of encoded/stringified query parameters, chances of breaking the URL is relatively high + URLs have char limits.
The API is a GET, so there's no way around it.
I'll defer to prod analytics since they heavily use this API, do they have an alternative?